### DISCUSSION TITLE
Issues with Assign

### DISCUSSION THREAD

**Discussion by fredrik-m-j** [prb28/vscode-amiga-assembly#266] [General]
Everything has been working perfectly but now I'm suddenly getting into a problem. I have had an assign set up so that my files are location independent and it has been working well for months. I don't know how to troubleshoot this.

I have the plugin installed + also WinUAE 4.10.0 (also tried upgrading to 4.10.1) on this pc.
I use 1.3 rom, and I have copied Assign (and CD and Dir) into \uae\dh0\c

This is my startup-sequence:
```
; Use assign for potential HD installation scenario
assign MyGame: SYS:

sys:main
; Quits the emulator at the end of the program
UAEquit
```

As said earlier - this has been working perfectly fine, but now I get: "Can't find SYS:" and "assign failed returncode 20"  in CLI
If I type the exact same thing into CLI it works, but then it's extra typing each time I change something in the code and run.
1> assign MyGame: SYS:

Any clues as to what could be wrong? Why is sys: only found when I type in CLI?

**Comment by prb28** [comment]
Did you check the encoding and line feed of your sartup-sequence?

**Reply by fredrik-m-j** [reply]
Aaaaaah, that was indeed the cause!
I brought the file into another OS 3.1-installation that I have and saved it with ed. Problem solved.
